include/sound_ids.h

```diff
@@ -12,648 +12,648 @@
+/* 0x00CC */ SOUND_VOICE_SMOKEY_SILVERCOINS1, // "Now try my new challenge"
+/* 0x00CD */ SOUND_VOICE_SMOKEY_SILVERCOINS2, // "Collect all the silver coins from my world"
+/* 0x00CE */ SOUND_VOICE_SMOKEY_GOODTRY, // "Excellent"
+/* 0x00CF */ SOUND_VOICE_SMOKEY_SILVERCOINS3, // "You must find 8 from each level and win"
+/* 0x00D0 */ SOUND_VOICE_SMOKEY_SILVERCOINS4, // "Then come back and see me"
+/* 0x00D1 */ SOUND_UNK_D1,
+/* 0x00D2 */ SOUND_VOICE_SMOKEY_BADLUCK, // "Bad luck"
+/* 0x00D3 */ SOUND_VOICE_SMOKEY_RETRY, // "Try again whenever you're ready"
+/* 0x00D4 */ SOUND_VOICE_SMOKEY_TROPHY1, // "You have earned a piece of the amulet"
+/* 0x00D5 */ SOUND_VOICE_SMOKEY_TROPHY2, // "Now try the trophy challenge"
+/* 0x00D6 */ SOUND_VOICE_SMOKEY_TROPHY3, // "Goodbye"
+/* 0x00D7 */ SOUND_VOICE_SMOKEY_REMATCH2, // "If you can beat me again"
+/* 0x00D8 */ SOUND_VOICE_SMOKEY_REMATCH3, // "I'll give you a special prize"
+/* 0x00D9 */ SOUND_VOICE_BUBBLER_ENCOUNTER1, // "Well done!"
+/* 0x00DA */ SOUND_VOICE_BUBBLER_ENCOUNTER2, // "Now I challenge you to a race!"
+/* 0x00DB */ SOUND_VOICE_BUBBLER_SILVERCOINS1, // "Try my new challenge"
+/* 0x00DC */ SOUND_VOICE_BUBBLER_SILVERCOINS2, // "Collect all the silver coins from my world."
+/* 0x00DD */ SOUND_VOICE_BUBBLER_SILVERCOINS3, // "Find 8 from each level and win"
+/* 0x00DE */ SOUND_VOICE_BUBBLER_BADLUCK, // "Tough luck, kid"
+/* 0x00DF */ SOUND_VOICE_BUBBLER_BADLUCK2, // "Maybe next time"
+/* 0x00E0 */ SOUND_VOICE_BUBBLER_RETRY, // "Come back whenever you're ready"
+/* 0x00E1 */ SOUND_VOICE_BUBBLER_TROPHY1, // "Well done, shrimp"
+/* 0x00E2 */ SOUND_VOICE_BUBBLER_TROPHY2, // "You have earned a piece of the amulet"
+/* 0x00E3 */ SOUND_VOICE_BUBBLER_TROPHY3, // "Now try the trophy challenge"
+/* 0x00E4 */ SOUND_VOICE_BUBBLER_TROPHY4, // "Goodbye"
+/* 0x00E5 */ SOUND_VOICE_BUBBLER_REMATCH2, // "If you can beat me again"
+/* 0x00E6 */ SOUND_VOICE_BUBBLER_REMATCH3, // "I'll give you a special prize"
+/* 0x00E7 */ SOUND_VOICE_BUBBLER_SILVERCOINS4, // "Then return to me"
+/* 0x00E8 */ SOUND_VOICE_BUBBLER_HAHA,
+/* 0x00E9 */ SOUND_VOICE_BLUEY_ENCOUNTER1, // "Well done"
+/* 0x00EA */ SOUND_VOICE_BLUEY_ENCOUNTER2, // "Now I challenge you to a race"
+/* 0x00EB */ SOUND_MENU_PICK2,
+/* 0x00EC */ SOUND_MENU_PICK3,
+/* 0x00ED */ SOUND_UNK_ED,
+/* 0x00EE */ SOUND_UNK_EE,
+/* 0x00EF */ SOUND_SELECT2,
+/* 0x00F0 */ SOUND_VOICE_TRICKY_WOAH3,
+/* 0x00F1 */ SOUND_VOICE_TRICKY_WOAH4,
+/* 0x00F2 */ SOUND_VOICE_SMOKEY_EH,
+/* 0x00F3 */ SOUND_VOICE_SMOKEY_HEH,
+/* 0x00F4 */ SOUND_VOICE_BLUEY_EH2,
+/* 0x00F5 */ SOUND_VOICE_BLUEY_OHNO2,
+/* 0x00F6 */ SOUND_VOICE_TRICKY_LAUGH,
+/* 0x00F7 */ SOUND_VOICE_TRICKY_LAUGH2,
+/* 0x00F8 */ SOUND_VOICE_SMOKEY_HAH,
+/* 0x00F9 */ SOUND_VOICE_SMOKEY_LAUGH,
+/* 0x00FA */ SOUND_VOICE_BLUEY_HAHA3,
+/* 0x00FB */ SOUND_VOICE_BLUEY_HAHA4,
+/* 0x00FC */ SOUND_UNK_FC,
+/* 0x00FD */ SOUND_CROWD,
+/* 0x00FE */ SOUND_MENU_PICK4,
+/* 0x00FF */ SOUND_UNK_FF,
+/* 0x0100 */  SOUND_VOICE_TT_LAP2,
```

Comment:
Looks like there's some indentation weirdness here and below.

Reply:
Fixed.